### DISCUSSION TITLE
unknown command "viz" for "linkerd"

### DISCUSSION THREAD

**Discussion by asencis** [linkerd/linkerd2#5961] [Q&A]
Hi, I've recently installed linkerd as per the instructions here: 

[https://linkerd.io/2.10/getting-started/](https://linkerd.io/2.10/getting-started/)

However, when I am attempting to use this command:

```bash
linkerd viz install | kubectl apply -f -
```

I receive the following error:

```
Error: unknown command "viz" for "linkerd"
```

I've installed linkerd via brew for MacOS...what have I not done or what have I done wrong? :)

Many thanks!

**Comment by wmorgan** [comment]
What is the output of `linkerd version` ? If it's 2.9 or below, Homebrew probably hasn't updated their package to 2.10. Either install the latest CLI directly, or use the earlier getting started guide here: https://linkerd.io/2.9/getting-started/

**Reply by asencis** [reply]
Client version: stable-2.9.1
Server version: stable-2.9.1

I guess I will need to wait for brew to update first.

**Reply by asencis** [reply]
@wmorgan What would be the easiest way to upgrade linkerd locally and then on the cluster? Is there a way to uninstall linkered from the kubernetes cluster, I don't want to assum it's just `kubectl delete linkerd`...

**Reply by wmorgan** [reply]
See https://linkerd.io/2.9/tasks/upgrade/ and https://linkerd.io/2.9/tasks/uninstall/ respectively.

Note that you will need the 2.10 CLI in order to upgrade to 2.10.
